doc/source/tutorial/examples/optimize_global_1.py

```diff
@@ -18,7 +17,7 @@ def eggholder(x):
 results['shgo'] = optimize.shgo(eggholder, bounds)
 results['DA'] = optimize.dual_annealing(eggholder, bounds)
 results['DE'] = optimize.differential_evolution(eggholder, bounds)
-results['BH'] = optimize.basinhopping(eggholder, bounds)
+results['BH'] = optimize.basinhopping(eggholder, x0=(0, 0))
```

Comment:
There was a refguide failure because `bounds` was 2D. Turns out `bounds` doesn't belong here, the second argument is the guess. `basinhopping` doesn't have a `bounds` parameter.